Library/Formula/xsd.rb

```diff
@@ -0,0 +1,78 @@
+class Xsd < Formula
+  desc "XML Data Binding for C++"
+  homepage "http://www.codesynthesis.com/products/xsd/"
+  url "http://www.codesynthesis.com/download/xsd/4.0/xsd-4.0.0+dep.tar.bz2"
+  sha256 "eca52a9c8f52cdbe2ae4e364e4a909503493a0d51ea388fc6c9734565a859817"
+  version "4.0.0"
+
+  depends_on "gcc" => :build
```

Comment:
Sorry, I forgot that. At the beginning, I was unable to get it to compile with clang++ and I went the gcc route until I got SIGSEGVs for mixing libc++ and libstdc++.